package.json

```
@@ -27,18 +28,26 @@
     "eslint-plugin-prefer-arrow": "^1.2.2",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
+    "esmify": "^2.1.1",
+    "fs-extra": "9.1.0",
+    "gulp": "4.0.2",
+    "gulp-concat": "2.6.1",
+    "gulp-terser": "2.0.1",
     "husky": "1.3.1",
     "jasmine": "^3.6.1",
     "jasmine-reporters": "2.3.2",
     "jasmine-spec-reporter": "4.2.1",
+    "npx": "10.2.2",
     "nyc": "15.0.1",
     "ts-node": "8.3.0",
     "tslint": "5.11.0",
     "tslint-config-standard": "8.0.1",
     "typescript": "3.9.6"
   },
   "scripts": {
-    "build": "tsc && copyfiles \"lib/**/*.json\" dist && copyfiles \"tests/**/*.js*\" dist && copyfiles \"package.json\" dist",
+    "build": "tsc && copyfiles \"lib/**/*.json\" dist && copyfiles \"tests/**/*.js*\" dist && copyfiles \"package.json\" dist && npm run wrap",
+    "browserify": "npx browserify -p esmify ./dist/lib/index.js --standalone RawIonSdk > dist/raw-sdk.js",
```

You should be able to just call this directly.

```suggestion
    "browserify": "browserify -p esmify ./dist/lib/index.js --standalone RawIonSdk > dist/raw-sdk.js",
```